internal/tools/go.mod

```diff
@@ -1,12 +1,178 @@
 module github.com/GoogleCloudPlatform/opentelemetry-operations-collector
 
-go 1.16
+go 1.18
 
 require (
-	github.com/blang/semver v3.5.1+incompatible // indirect
 	github.com/client9/misspell v0.3.4
-	github.com/golangci/golangci-lint v1.42.0
+	github.com/golangci/golangci-lint v1.48.0
 	github.com/google/addlicense v1.0.0
 	github.com/google/googet v2.13.0+incompatible
 	github.com/pavius/impi v0.0.3
 )
+
+require (
```

Comment:
Ran a `go mod tidy` on this module. Not sure if that's something we care to do since it's tooling code.

Reply:
This is fine imo